Flatten/Flatten/MainPage.xaml

```diff
@@ -0,0 +1,18 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:pv="clr-namespace:Syncfusion.Maui.PdfViewer;assembly=Syncfusion.Maui.PdfViewer"
+             x:Class="Flatten.MainPage">
+
+    <Grid RowDefinitions="50,*">
+        <pv:SfPdfViewer x:Name="pdfViewer" Grid.Row="1"/>
+        <HorizontalStackLayout HorizontalOptions="Start">
+            <Button Text="&#xe75f;" 
```

Comment:
<img width="389" alt="image" src="https://github.com/SyncfusionExamples/maui-pdf-viewer-examples/assets/65169077/ae6df6aa-551e-496d-84b9-f958a78dc51c">

Please name the events based on purpose and also as we discussed, modify the UI

Reply:
addressed